builderator.gemspec

```diff
@@ -24,8 +24,9 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'thor-scmversion', '1.7.0'
 
   spec.add_dependency 'aws-sdk', '~> 2.0'
-  spec.add_dependency 'berkshelf', '~> 3.2'
-  spec.add_dependency 'chef', '~> 12.0'
+  spec.add_dependency 'berkshelf', '~> 4.3'
+  spec.add_dependency 'dep_selector', '~> 1.0'
+  # spec.add_dependency 'chef', '~> 12.5'
```

Comment:
Only in one unused module right now. I'm leaving it out until we actually need it.